other/management/commands/clean-notifications.py

```diff
@@ -0,0 +1,23 @@
+"""Clean up old notifications (older than 3 months)"""
+from datetime import timedelta
+import json
+
+from django.utils.timezone import now
+from django.core.management.base import BaseCommand
+from notifications.models import Notification
+
+class Command(BaseCommand):
+    help = 'Clean up old notifications (older than 3 months)'
+
+    def add_arguments(self, parser):
+        parser.add_argument('path', type=str, help='Give directory path for dumping old notifs without end slash')
+
+    def handle(self, *args, **options):
+        file_path = options['path'] + '/' + str(now().date()) + '.txt'
```

Comment:
I'll suggest append it to the same file and logrotate instead, cause this will create one file daily. You can probably logrotate once a month or something. Just a suggestion though.